packages/modules/sma_sunny_boy/device.py

```diff
@@ -14,7 +14,7 @@
 from modules.sma_sunny_boy.config import (SmaSunnyBoy, SmaSunnyBoyBatSetup, SmaSunnyBoyConfiguration,
                                           SmaSunnyBoyCounterSetup, SmaSunnyBoyInverterConfiguration,
                                           SmaSunnyBoyInverterSetup, SmaSunnyBoySmartEnergyBatSetup)
-from modules.sma_sunny_boy.inverter_version import SmaInverterVersion
+from modules.sma_sunny_boy.inv_version import SmaInverterVersion
```

Comment:
Warum wird hier abgekürzt? Die paar Byte müssen wir doch nicht einsparen. Wenn es eine `inverter.py` gibt, warum nicht auch eine `inverter_version.py`? Dann sind auch die Zusammenhänge besser erkennbar.

Reply:
Notiz: der Dateiname wird für die Liste der konfigurierbaren Komponenten in 2.0 genutzt.